broker-cli/scripts/postinstall.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/sh
 
+# Setup auto-completion for bash
+if [-f ~/.bashrc]; then
+    echo "source <(sparkswap completion bash)" >> ~/.bashrc && source ~/.bashrc
```

Comment:
This line will add that text each time `npm i` is ran, which is not necessarily what we want. Instead of adding the line automatically in the script, we should prompt the use to add it manually IF they want auto completion.